components/src/core/components/CardTable/Cell/ProfilePicture.vue

```diff
@@ -12,6 +12,8 @@
     :link="profilePicture.link"
     :imageSrc="profilePicture.src"
     :link-mode="profilePicture.target"
+    :isCustomFunctionExist="isCustomFunctionExist"
```

Comment:
This is not the best way to do this. consider this approach.

Inside `components/CardTable/Cell/ProfilePicture.vue`

```html
<template>
...
  <oxd-profile-pic
    v-else
    :size="profilePicture.size"
    :link="profilePicture.link"
    :image-src="profilePicture.src"
    :link-mode="profilePicture.target"
    :link-handler="handleLinkClick"
  />
...
</template>
```

```ts
<script lang="ts">
...
   setup() {
     ...
     const handleLinkClick = (event: MouseEvent) => {
        const cellConfig = props.header?.cellConfig;
        if (cellConfig && typeof cellConfig?.onClick === 'function') {
          event.preventDefault();
          props.header?.cellConfig.onClick(props.rowItem, event);
        }
    };
     ...
   }
....
</script>
```

Inside `components/ProfilePic/ProfilePic.vue`

```ts
<script lang="ts">
...
export default defineComponent({
  props: {
    linkHandler: {
      type: Function,
      required: false
    }
  }
})
...
methods: {
    handleLinkClick(event: MouseEvent) {
      if (this.linkHandler && typeof this.linkHandler === "function") {
        this.linkHandler(event);
      }
    },
  }
...
```